src/GraphQLParser.sln

```diff
@@ -15,9 +15,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		..\.gitignore = ..\.gitignore
 		Directory.Build.props = Directory.Build.props
 		Directory.Build.targets = Directory.Build.targets
+		graphql.snk = graphql.snk
```

Comment:
private.snk ?

Reply:
I thought some more on this. `graphql.snk` is fine since we may copy-paste this file across all org repos.

Reply:
that was exactly my thinking when naming it. I've used the same snk for signing in https://github.com/graphql-dotnet/graphql-dotnet/pull/3265 - unsure if there is some smarter way to share it between repos without much effort. 

Assemblies could of course just be signed in gh action rather than during build but then making "custom" builds from source would become a real pain for everyone.